pyfr/mixins/elements.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+from pyfr.shapes import BaseShape
+from pyfr.util import memoize, subclass_where
+
+class ElementOpsMixin:
```

Comment:
Keep these mixins close to home rather than in a generic dumping-ground folder.  (A bit like how the fluids mixin is in the Euler elements file.)

Reply:
These mixins get used in multiple places hence why they are in one place. For exmaple this one gets used in the ascent plugin but also in the vtk writter.

Reply:
Right, but it really belongs and originates in the VTK writer (Ascent is a plugin so it is of lesser importance).